.expeditor/verify.pipeline.yml

```diff
@@ -573,6 +573,18 @@ steps:
 # still build. - Linux
 #######################################################################
 
+  - label: "[build] :linux: test-probe"
+    env:
+      HAB_LICENSE: "accept-no-persist"
+    command:
+      - hab pkg build -D test-services/test-probe
```

Comment:
This will put you in a docker-in-docker situation.  I'd recommend removing the `-D` here and using 
```
expeditor:
  executor:
    docker:
      privileged: true
```

Or, you can use a linux executor, but it's probably a better idea to use the docker executors unless otherwise needed.

Reply:
:man_facepalming: Good call.